packages/retag/src/index.ts

```diff
@@ -47,25 +41,21 @@ async function main() {
  * This shouldn't normally need to run, since we run `tagSingle` whenever we publish a package.
  * But this should be run if the way we calculate tags changes (e.g. when a new release is allowed to be tagged "latest").
  */
-async function tag(dry: boolean, nProcesses: number, name?: string) {
+async function tag(dry: boolean, _nProcesses: number, name?: string) {
```

Comment:
Should this process stuff also go away or are we trying to keep this API?

Reply:
I missed this one, removed most of it.